spelling error in blackmagic/Makefile

depricated->deprecated

``
Makefile build system is depricated.

Please use Meson build system instead.

For most basic build, which builds only Black Magic Debug Application (BMDA), run the following commands:
> meson setup build
> meson compile -C build

You can find example firmware configuration files in the `cross-file` subdirectory.
For example, to build a firmware for the native hardware run the following commands:
> meson setup build-native --cross-file cross-file/native.ini --werror
> meson compile -C build-native

For further instructions please refer to the README.md in the root directory of this repository.

make: *** [all] Error 1

``

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 all:
-	@echo "Makefile build system is depricated."
+	@echo "Makefile build system is deprecated."
 	@echo ""
 	@echo "Please use Meson build system instead."
 	@echo ""

